api/src/main/java/com/codeforcommunity/rest/subrouter/AuthRouter.java

```diff
@@ -88,4 +101,28 @@ private void handlePostNewUser(RoutingContext ctx) {
 
     end(ctx.response(), 201, JsonObject.mapFrom(response).toString());
   }
+
+  private void handleVerifySecretKey(RoutingContext ctx) {
+    String secret = ctx.pathParam("secret_key");
+
+    try {
+      authProcessor.validateSecretKey(secret);
+      end(ctx.response(), 200);
+    }
+    catch (AuthException e) {
```

Comment:
Same issue as below you need to follow how we're handling exceptions now. See Luke's recently merged changes for how exceptions should be handled (better yet reach out to him / me about it)